econsa/correlation.py

```diff
@@ -10,7 +10,7 @@
 from statsmodels.stats.correlation_tools import corr_nearest
 
 
-def gc_correlation(marginals, corr, force_calc=False, rule="sobol", num_points=1000):
+def gc_correlation(marginals, corr, force_calc=False, order=15):
```

Comment:
Please switch order of `force_calc` and `order` arguments in all functions where they show up together.